exercises/list-ops/src/test/kotlin/ListExtensionsTest.kt

```diff
@@ -0,0 +1,141 @@
+import org.junit.Ignore
+import org.junit.Test
+import kotlin.test.assertEquals
+
+/*
+ * version: 1.0.0
```

Comment:
`xjava` does not follow the canonical test suite (I suspect it may predate it) so I went ahead and wrote the Kotlin tests and example implementation fresh. I can port to `xjava` once we've ironed out any kinks!